docker-compose.yml

```diff
@@ -50,6 +50,8 @@ services:
     expose: ["5000"]
     networks:
      - internal
+    entrypoint: ""
+    command: "./runserver.sh"
```

Comment:
The Dockerfile defines the entrypoint as `flask`, and the command as `run`, so this overrides it to the shell script so that cli commands get run. I like having the entrypoint be flask, because it means that I can do things like `docker run research-app-api some-command` and call flask commands.